hypertension/.vscode/launch.json

```diff
@@ -1,5 +1,29 @@
 {
     "configurations": [
+        {
+            "type": "aws-sam",
+            "request": "direct-invoke",
+            "name": "hypertension:VroMainFunction",
+            "invokeTarget": {
+                "target": "template",
+                "templatePath": "${workspaceFolder}/template.yaml",
+                "logicalId": "VroMainFunction"
+            },
+            "lambda": {
+                "payload": {
+                    "path": "/Users/julialallen/workspace/Virtual-Regional-Office/hypertension/vro-main/pdf-event.json"
+                },
+                "environmentVariables": {
+                    "WKHTMLTOPDF_PATH": "/opt/bin/wkhtmltopdf",
+                    "FONTCONFIG_PATH": "/opt/fonts",
+                    "STAGE": "dev",
+                    "LighthouseTokenUrl": "!Ref LighthouseTokenUrl",
+                    "LighthouseClientId": "!Ref LighthouseClientId",
+                    "LighthouseJwtAud": "!Ref LighthouseJwtAud",
+                    "LighthousePrivateRsaKeySecretArn": "!Ref VroLhPrivateRsaKey"
+                }
+            }
+        },
         {
```

Comment:
I added my local debug config for the VroMain function since this file was already checked in.